src/nodes/matmulinteger.h

```diff
@@ -49,6 +49,7 @@ class MatMulInteger : public Node {
 	{
 		std::string intype = A->data_type_str();
 		std::string outtype = Y->data_type_str();
+        std::string weighttype = B->data_type_str();
```

Comment:
The indentation here is with spaces, could I ask you to fix to tabs.

Reply:
The latest commit has tabs - please check.